lib/api.js

```diff
@@ -653,6 +658,30 @@ api.declare({
   return res.reply(amiUsage);
 });
 
+/**
+ * Lists current EBS volume usage by returning a list of objects in the form:
+ * {
+ *  region: string,
+ *  volumetype: string,
+ *  state: string,
+ *  totalcount: string,
+ *  totalgb: string,
+ *  touched: timestamp,
+ * }
```

Comment:
You might also mention that there is one row per region+volumetype+state, and that `touched` is the last time that information was updated.